lib/css_parser/parser.rb

```diff
@@ -113,7 +114,7 @@ def find_rule_sets(selectors, media_types = :all)
     #   parser = CssParser::Parser.new
     #   parser.add_block!(css)
     def add_block!(block, options = {})
-      options = {:base_uri => nil, :base_dir => nil, :charset => nil, :media_types => :all, :only_media_types => :all}.merge(options)
+      options = {:ignore_import => nil, :allow_font_face => nil, :base_uri => nil, :base_dir => nil, :charset => nil, :media_types => :all, :only_media_types => :all}.merge(options)
```

Comment:
> Can you use the new hash syntax?

I'd disagree with this. Using the style that the file is currently in I find to be a better approach.